k8s/aat/common/admin/check-acr-sync-cronjob.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: check-acr-sync
+  namespace: admin
+  labels:
+    app: check-acr-sync
+spec:
+  template:
+    metadata:
+      labels:
+        app: check-acr-sync
+    spec:
+        restartPolicy: Never
+        containers:
+        - name: check-acr-sync
+          image: hmctspublic.azurecr.io/check-acr-sync:db1rse
+          imagePullPolicy: Always
```

Always? wouldn't this be better?
```suggestion
          imagePullPolicy: IfNotPresent
```